CONTRIBUTING.md

```diff
@@ -0,0 +1,154 @@
+should take place downstream first in order to ensure new dependencies are 
+available.  Changes made to componenet models that change calls in the 
+FMScoupler must include changes to corresponding null/data models in order to 
+maintain the automated build tests.
+
+Please keep the changes in a single pull to be as small as possible to help
+reviewer(s) quickly evaluate changes.  If you do have a large update, try to
+split the update into small, logical pull requests.  If the update includes code
+refactoring, please submit a pull request that includes just the code refactoring.
+
+Once a pull request is created, a member of the Modeling Systems group will
+review the changes, and, if necessary, will work with the author of the pull
+request to modify their code changes. Note that merging pull requests is
+contingent on several factors, including the ability to support the code changes
+long-term, portability, and the scope of the impact on the code base. Therefore,
+Modeling Systems does not guarantee that all pull requests will be accepted,
+even if the changes pass the initial testing phases, and are otherwise correct.
+
+## Reviewing Pull Requests
+
+When reviewing a pull request, members of MSD should look for the following:
+
+- Design
+  - Does the code change belong in the FMScoupler library or does it better belong elsewhere such as a component repository or FMS?
+  - Could existing routines/modules be utilized to reduce redundancy?
+  - Temporary changes/fixes meant to be removed should be avoided whenever possible
+- Functionality
+  - Does this PR do what is intended (and stated)
+  - Are the changes good for both end-users and developers?
+  - Will the code change impact existing end-users needlessly?
+- Complexity
+  - Are the changes easily understood by the reader / reviewer?
+- Testing
+  - Code changes should include a test program or a modification to a test program to ensure the code is covered by the test suite
+- Comments
+  - Inline comments for complex code segments or intricacies to make the purpose of the code reasonably clear
+- Style and Consistency
+  - Code should follow the [syle guide](CODE_STYLE.md) in general, but should also be consistent to the file the change is made in
+- Documentation
+  - If a PR changes the behaviour or instructions, accompanying documentation should also change
+- Thoroughness
+  - Reviews should be done line by line, and the surrounding context/file should be taken into account
+
+Comments on pull requests should be courteous and constructive, giving useful feedback and explanations for why changes should be made. See the [code of conduct](CODE_OF_CONDUCT.md) for more information.
+
+## Tests
+
+FMScoupler uses GitHub Actions to test FMScoupler.  Users may create
+unit tests, code coverage tests, and regression tests for new and existing code
+in yaml (.yml) files.  Github provides a guide
+(https://help.github.com/en/articles/about-continuous-integration) for
+implementing CI tests.
```

Comment:
Might be good to mention the null_model script in this section, since thats the only real test we use in this repository besides our alpha/beta testing.